CodeForAfrica/ckanext-embed
# Dataset Embed

#### What does this PR do?
Enable organizations to embed their datasets on their websites as stored on CodeForAfrica

#### Description of Task to be completed?
- As an organization, we should be able to embed our datasets hosted on open africa on our own website
- As an organization, we should be able to have visitors to our website search through our datasets as hosted on OpenAfrica

#### Screenshots
View of list of datasets:
![image](https://user-images.githubusercontent.com/8082197/33200050-0989ddda-d104-11e7-8fd1-f7eec2661f9e.png)
View of a single dataset's details:
![image](https://user-images.githubusercontent.com/8082197/33200047-05f15cc0-d104-11e7-8acc-1ae12f58f432.png)
View of a single dataset's resources:
![image](https://user-images.githubusercontent.com/8082197/33211055-d0d36c3e-d12d-11e7-9f16-ac41c3d2a203.png)
View of a single dataset's activity:
![image](https://user-images.githubusercontent.com/8082197/33200010-dcec9592-d103-11e7-87f6-d1f538995d10.png)
#### How can this be manually tested

After the extension is successfully installed on a running CKAN instance:

* Create an organization and add public datasets to it

* Create an iframe on another site or HTML page as follows:
```
<iframe
  width="600"
  height="450"
  frameborder="0" style="border:0"
  src="https://<ckan-url>/embed/?organization=<organization-name>">
</iframe>
```
* Feel free to customize the iframe properties

>**Note**: The `<organization-name>` is the one that appears on the organization URL


* If everything is set up successfully, the datasets you created should appear within the iframe as shown in the screenshots above
